### DISCUSSION TITLE
GitHub Actions Cache usage

### DISCUSSION THREAD

**Discussion by ohmer** [community/community#25237] [Actions]
<p>Hi there,</p>
<p>Is there a way to access the GitHub Actions Cache usage?</p>
<p>Cheers,<br>
<a class="mention" href="/ohmer">@ohmer</a></p>

**Comment by ashwinsangem** [comment, marked as answer]
<p>Yes, you can do so by using these APIs: <a href="https://docs.github.com/en/rest/reference/actions#cache" class="inline-onebox" rel="noopener nofollow ugc">Actions - GitHub Docs</a></p>
